src/NanoporeMultiEventalign.jl

```diff
@@ -4,17 +4,16 @@ module NanoporeMultiEventalign
 using BioSequences
 using FASTX
 using Plots
-
 using HDF5
-using BioSequences
-using FASTX
+using DynamicAxisWarping
 
 include("utils.jl")
 include("persistence.jl")
 include("dtw.jl")
 
 
-export dtw
+export dtw_test
```

Comment:
dtw_test was just a function I added to make you hello world, you can remove it entirely